cybergis_compute_client/CyberGISCompute.py

```diff
@@ -212,6 +218,25 @@ def create_job(self, maintainer='community_contribution', hpc=None, hpcUsername=
         self.login()
         return Job(maintainer=maintainer, hpc=hpc, id=None, hpcUsername=hpcUsername, hpcPassword=hpcPassword, client=self.client, isJupyter=self.isJupyter, jupyterhubApiToken=self.jupyterhubApiToken, printJob=verbose)
 
+    def run_job_using_params(self,
+                             input_params=[],
+                             maintainer='community_contribution',
+                             hpc="keeling_community",
+                             hpcUsername=None,
+                             hpcPassword=None,
+                             localExecutableFolder={"type": "git",
+                                                    "gitId": "hello_world"},
```

Comment:
It seems this is how the user would pass in the "job"/"model" which will probably not be intuitive to them.

Reply:
Yup I will handle that in ParamAccumulator (Will maybe change the name to something else since its handling "job"/"mode" preprocessing as well).